util/codec/codec.go

```diff
@@ -1073,3 +1073,92 @@ func appendFloatToChunk(val float64, chk *chunk.Chunk, colIdx int, ft *types.Fie
 		chk.AppendFloat64(colIdx, val)
 	}
 }
+
+// EncodeColumnTo encodes each row of this column and append encoded data into buf.
+func EncodeColumnTo(sc *stmtctx.StatementContext, n int, col *chunk.Column, buf [][]byte, ft *types.FieldType) ([][]byte, error) {
```

Comment:
After investigating, we update the way to encode our data.
And the benchmark result has been updated, PTAL @XuHuaiyu